src/features/theme-switcher/ui/ThemeSwitcher.tsx

```diff
@@ -1,18 +1,19 @@
 import { Space, Switch } from "antd";
 import { useUnit } from "effector-react";
 import { $theme, themeChanged } from "../model";
+import './ThemeSwitcher.css'
 
 export const ThemeSwitcher = () => {
   const [themeValue, themeChangedFn] = useUnit([$theme, themeChanged]);
 
   const onChange = (checked: boolean) => {
-    checked ? themeChangedFn("dark") : themeChangedFn("light");
+    checked ? themeChangedFn("DARK") : themeChangedFn("LIGHT");
```

Comment:
checked ? themeChangedFn(themes.DARK) : themeChangedFn(themes.LIGHT);

Reply:
onChange should be replaced by handleChange
also use effector.prepend for Events https://effector.dev/docs/api/effector/event/#prependfn

It is better to declare handlers outside the component using prepends